core/storage/base_model/gae_models.py

```diff
@@ -240,6 +240,21 @@ class BaseCommitLogEntryModel(BaseModel):
     # The version number of the model after this commit.
     version = ndb.IntegerProperty()
 
+    @classmethod
+    def _get_instance_id(cls, instance_id, version):
```

Comment:
It is confusing to pass in an instance_id to a method called get_instance_id() :)

Consider renaming the first argument to target_id or target_entity_id (since that entity was the target of the commit).

Reply:
Done, renamed it.